src/components/tooltips/line-popover.js

```diff
@@ -0,0 +1,290 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Box from '@mui/material/Box';
+import LinearProgress from '@mui/material/LinearProgress';
+import {
+    Paper,
+    Popover,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+} from '@mui/material';
+import { useIntl } from 'react-intl';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import { fetchLine } from '../../utils/rest-api';
+import { useSelector } from 'react-redux';
+import { RunningStatus } from '../utils/running-status';
+
+const LinePopover = ({ studyUuid, anchorEl, lineId, loadFlowStatus }) => {
+    const currentNode = useSelector((state) => state.currentTreeNode);
+    const [lineInfo, setLineInfo] = useState(null);
+    const intl = useIntl();
+
+    useEffect(() => {
+        if (lineId !== '') {
+            fetchLine(studyUuid, currentNode.id, lineId).then((value) => {
+                setLineInfo(value);
+            });
+        } else {
+            setLineInfo(null);
+        }
+    }, [lineId, currentNode.id, studyUuid]);
+
+    const handlePopoverClose = () => {
+        setLineInfo(null);
+    };
+
+    const checkValue = (value) => {
+        if (value) {
+            return value;
+        } else {
+            return '';
+        }
+    };
```

Comment:
you can change it to (remove else)
const checkValue = (value) => {
        if (value) {
            return value;
        } 
        return '';
    };